site/dat/repo/blazemeter.json

```diff
@@ -732,5 +732,26 @@
         "downloadUrl": "https://github.com/Blazemeter/SiebelPlugin/releases/download/v0.1.0-beta/jmeter-bzm-siebel-correlation-0.1.0-beta.jar"
       }
     }
+  },
+  {
+    "id": "mqmeter",
+    "name": "IBM MQ Support",
+    "description": "A JMeter Plugin to put and get message on IBM MQ Queue, also publish message on Topic. It connect to MQ Server through server channel using ip address, port number, userID and password (if the channel has CHLAUTH rules).",
+    "screenshotUrl": "https://github.com/JoseLuisSR/img2/blob/master/mqmeter/MQClientSampler.png",
+    "helpUrl": "https://github.com/JoseLuisSR/mqmeter",
+    "vendor": "JoseLuisSR",
+    "markerClass": "co.signal.mqmeter.MQClientSampler",
+    "componentClasses": [
+      "co.signal.mqmeter.MQPublishSampler",
+	  "co.signal.mqmeter.MQClientSampler"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs. (no-mixed-spaces-and-tabs)](https://app.codacy.com/app/apc4/jmeter-plugins/pullRequest?prid=4661576)